components/Organization/OrganizationList.tsx

```diff
@@ -21,7 +21,7 @@ export class OrganizationCardList extends ScrollList<OrganizationListProps> {
   static Layout = ({ value = [] }: OrganizationListProps) => (
     <div>
       {value.map(item => (
-        <Row key={item.id}>
+        <Row key={item.id} style={{ padding: '.5rem' }}>
```

Comment:
```suggestion
        <Row key={item.id} className="p-2">
```
https://getbootstrap.com/docs/5.2/utilities/spacing/#margin-and-padding